kit/metrics/cloudwatch2: fails to build ... again

Cloudwatch2 metrics are failing to build again with this message:
```
$go get -t github.com/go-kit/kit/...
# github.com/go-kit/kit/metrics/cloudwatch2
metrics/cloudwatch2/cloudwatch2.go:35:24: undefined: cloudwatchiface.CloudWatchAPI
```
This is due to an update in the the cloudwatch api: https://github.com/aws/aws-sdk-go-v2/pull/298

--- a/metrics/cloudwatch2/cloudwatch2.go
+++ b/metrics/cloudwatch2/cloudwatch2.go
@@ -32,7 +32,7 @@ type CloudWatch struct {
 	mtx                   sync.RWMutex
 	sem                   chan struct{}
 	namespace             string
-	svc                   cloudwatchiface.CloudWatchAPI
+	svc                   cloudwatchiface.ClientAPI
 	counters              *lv.Space
 	logger                log.Logger
 	numConcurrentRequests int
@@ -66,7 +66,7 @@ func WithConcurrentRequests(n int) Option {
 // Namespace is applied to all created metrics and maps to the CloudWatch namespace.
 // Callers must ensure that regular calls to Send are performed, either
 // manually or with one of the helper methods.
-func New(namespace string, svc cloudwatchiface.CloudWatchAPI, options ...Option) *CloudWatch {
+func New(namespace string, svc cloudwatchiface.ClientAPI, options ...Option) *CloudWatch {
 	cw := &CloudWatch{
 		namespace:             namespace,
 		svc:                   svc,

--- a/metrics/cloudwatch2/cloudwatch2_test.go
+++ b/metrics/cloudwatch2/cloudwatch2_test.go
@@ -73,7 +73,7 @@ func TestStats(t *testing.T) {
 }
 
 type mockCloudWatch struct {
-	cloudwatchiface.CloudWatchAPI
+	cloudwatchiface.ClientAPI
 	latestName string
 	latestData []cloudwatch.MetricDatum
 }

